contrib/hack/kubectl.sh

```diff
@@ -17,6 +17,12 @@ set -e
 
 ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)"
 
+if [[ "${WITH_TTY:-}" -eq 1 ]]; then
```

Comment:
Along those line, I'm NOT LGTM on renaming this kubectl.sh - I want people to be able to add contrib/hack to their path and then run `kubectl` from whatever scripts they might have.